src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutorExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Triggers;
+
+namespace Microsoft.Azure.WebJobs.Host.Executors
+{
+    /// <summary>Provides extension methods for the <see cref="IFunctionExecutor"/> interface.</summary>
+    public static class FunctionExecutorExtensions
```

Comment:
Make this internal, then the method below can be public. We don't want this extension in our public surface area.